errors.go

```diff
@@ -99,20 +99,30 @@ func (e *e) Cause() error {
 	return e.cause
 }
 
-// Wrap returns an error annotating the cause with message.
-// If cause is nil, Wrap returns nil.
-func Wrap(cause error, message string) error {
+func wrap(cause error, message string) error {
```

Comment:
This change is bigger than it needs to be. I suggest copying Wrap to Wrapf and changing the call to errors.New with a fmt.Errorf